examples/react-native/__tests__/intro.test.js

```diff
@@ -31,11 +31,15 @@ it('renders the ActivityIndicator component', () => {
 });
 
 it('renders the Image component', done => {
-  Image.getSize('path.jpg', (width, height) => {
-    const tree = renderer.create(<Image style={{height, width}} />).toJSON();
-    expect(tree).toMatchSnapshot();
-    done();
-  });
+  Image.getSize(
+    'path.jpg',
+    (width, height) => {
+      const tree = renderer.create(<Image style={{height, width}} />).toJSON();
+      expect(tree).toMatchSnapshot();
+      done();
+    },
+    done
```

Comment:
this now errors (and times out since we never call `done`), so I added this. It'll fail, this is the full error:

```
TypeError: Invalid attempt to destructure non-iterable instance.
In order to be iterable, non-array objects must have a [Symbol.iterator]() method.
    at _nonIterableRest (/Users/simen/repos/jest/node_modules/react-native/Libraries/Image/Image.ios.js:24:775)
    at _slicedToArray (/Users/simen/repos/jest/node_modules/react-native/Libraries/Image/Image.ios.js:24:720)
    at /Users/simen/repos/jest/node_modules/react-native/Libraries/Image/Image.ios.js:34:37
    at tryCallOne (/Users/simen/repos/jest/node_modules/react-native/node_modules/promise/lib/core.js:37:12)
    at /Users/simen/repos/jest/node_modules/react-native/node_modules/promise/lib/core.js:123:15
    at flush (/Users/simen/repos/jest/node_modules/asap/raw.js:50:29)
    at processTicksAndRejections (internal/process/task_queues.js:76:11)
```

Reply:
Prior to RN 0.62, `Image.getSize` used `NativeModules.ImageViewManager` on iOS (and `NativeModules.ImageLoader` for Android) under the hood, which was mocked in react-native preset: https://github.com/facebook/react-native/blob/caa7829aac01256a1f01dfda73f7d0a5c0371064/jest/setup.js#L214-L223. 

With https://github.com/facebook/react-native/commit/d37baa78f11f36aa5fb84307cc29ebe2bf444a33, it was replaced by `NativeImageLoaderIOS` which doesn't have a mock. cc @cpojer

Reply:
We are doing some funky stuff internally, that's probably why we didn't notice. I recommend changing the example to something else.